evhtp.c

```diff
@@ -2558,15 +2583,18 @@ htp__connection_resumecb_(int fd, short events, void * arg)
 {
     evhtp_connection_t * c = arg;
 
+    log_debug("resumecb");
+
     /* unset the pause flag */
     HTP_FLAG_OFF(c, EVHTP_CONN_FLAG_PAUSED);
 
-    if (c->request)
-    {
+    if (c->request) {
+        log_debug("cr status = OK %d", c->cr_status);
         c->cr_status = EVHTP_RES_OK;
```

Comment:
Not quite sure how this affects things. We assume that resumecb runs BEFORE anything.